XDMA/linux-kernel/xdma/alinx_ptp.h

```diff
@@ -1,19 +1,19 @@
-#ifndef EXAMPLE_PTP_H
-#define EXAMPLE_PTP_H
+#ifndef ALINX_PTP_H
+#define ALINX_PTP_H
 
 #include <linux/ptp_clock_kernel.h>
 #include "xdma_mod.h"
 #include "libxdma.h"
 
-#define NEXT_PULSE_AT_HI (0x002c)
-#define NEXT_PULSE_AT_LO (0x0030)
-#define CYCLE_1S (0x0034)
-#define SYS_CLOCK_HI (0x0380)
-#define SYS_CLOCK_LO (0x0384)
+#define NEXT_PULSE_AT_HI 0x002c
+#define NEXT_PULSE_AT_LO 0x0030
+#define CYCLE_1S 0x0034
+#define SYS_CLOCK_HI 0x0380
+#define SYS_CLOCK_LO 0x0384
```

Comment:
These are register memory offset. Not real value.
How about this?
```suggestion
#define REG_NEXT_PULSE_AT_HI 0x002c
#define REG_NEXT_PULSE_AT_LO 0x0030
#define REG_CYCLE_1S 0x0034
#define REG_SYS_CLOCK_HI 0x0380
#define REG_SYS_CLOCK_LO 0x0384
```